Documentation/SystemRequirements/PHP.rst.txt

```diff
@@ -47,6 +46,111 @@ Depending on the use case, the following extensions may also be required:
 * **openssl** (OpenSSL is required for sending SMTP mails over an encrypted channel endpoint)
 * **intl** (when using unicode-based filesystems)
 
+Opcache settings
+----------------
+
+It is recommended to enable opcache and tune the settings for optimal
+performance. Most settings will already have a good default value, but
+the settings will also depend on number of scripts, available memory etc.
+
+.. warning::
+
+   Opcache must be adapted to your specific needs. Some of the settings which
+   improve performance should not be activated for TYPO3 or come with a risk.
+   The following information has been put together carefully but it is recommended
+   to double-check the given recommendations regularly. Please help and contribute
+   to this page if you find something which is not correct!
+
+   In particular, the following are not recommended for TYPO3:
+
+   *  opcache.save_comments=0
+
+We add information which is TYPO3 specific for some of the settings but will not
+cover all settings. Please refer to the
+`official PHP documentation <https://www.php.net/manual/en/opcache.configuration.php>`__
+as well.
+
+.. code-block:: ini
+   :caption: php.ini
+
+   opcache.enable=1
+   opcache.revalidate_freq=30
+   opcache.revalidate_path=0
+
+.. confval:: opcache.enable
+
+   :Default: =1
+   :Faster: =1
```

Comment:
What does "Faster" mean?

Reply:
It mean the setting 1 will typically be faster than the setting 0. However, this looked better in the table I had used previously. I will push another change.